src/engraving/rendering/score/slurtielayout.h

```diff
@@ -19,12 +19,13 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-#ifndef MU_ENGRAVING_SLURTIELAYOUT_DEV_H
-#define MU_ENGRAVING_SLURTIELAYOUT_DEV_H
+#pragma once
```

Comment:
Nothing bad about this, but since we use the #ifndef / #endif for every other header file in our repo, I'd rather keep it consistent.

Reply:
This is changing; we're now gradually switching to `#pragma once` everywhere. New files should use `#pragma once`, and existing files could/should be updated when they are touched anyway.